topdown/strings.go

```diff
@@ -227,6 +227,36 @@ func builtinReplace(a, b, c ast.Value) (ast.Value, error) {
 	return ast.String(strings.Replace(string(s), string(old), string(new), -1)), nil
 }
 
+func builtinReplacerReplace(a, b ast.Value) (ast.Value, error) {
+	asJSON, err := ast.JSON(a)
+	if err != nil {
+		return nil, err
+	}
+	oldnewObj, ok := asJSON.(map[string]interface{})
+	if !ok {
+		return nil, builtins.NewOperandTypeErr(1, a, "object")
+	}
+
+	s, err := builtins.StringOperand(b, 2)
+	if err != nil {
+		return nil, err
+	}
+
+	var oldnewArr []string
+	for k, v := range oldnewObj {
+		strVal, ok := v.(string)
+		if !ok {
+			return nil, fmt.Errorf("non-string value included in object: %s", a.String())
```

Comment:
`builtins.NewOperandTypeErr` didn't seem to make sense here so I've created a new error here.

Reply:
Understood. I've made the changes here:

https://github.com/open-policy-agent/opa/pull/1825/files#diff-63b9886aafbee520e97e3ddd5138d87cR250